city.py

```diff
@@ -5,7 +5,7 @@
 
 
 @dataclass
-class Game:
+class City:
     citizens: List['Citizen'] = field(default_factory=list)
```

Comment:
Please move action_manager to game_controller. It should not be a Part of the city

Reply:
Done